lib/code_corps/stripe_service/adapters/stripe_connect_account.ex

```diff
@@ -93,17 +125,31 @@ defmodule CodeCorps.StripeService.Adapters.StripeConnectAccountAdapter do
     |> Map.take(@non_stripe_attributes)
   end
 
+  defp remove_read_only_values(attributes, [head | tail]) do
+    remove_read_only_values(attributes, head)
+    remove_read_only_values(attributes, tail)
```

Comment:
How does this work? Seems to me like it only returns the result from removing read only values from the final item in the list (after it recursively gets through the tail). Are we missing some chaining here?